### PR TITLE
Update Docker Compose to connect with local native Ollama installation.

### PR DESCRIPTION
**Pull request by jameskingtech**
Instead of deploying Ollama through Docker and being limited to CPU only, this docker compose will connect directly with a native MacOS installation of Ollama.

### CODE REVIEW

**Comment by jameskingtech** [comment]
Might be worth bringing this into a different branch than main so that it's an option people can use instead if preferred.

I've used this locally on my MacBook M1 Pro and got good performance, everything is hooked up with OpenWebUI with a few different Llama models installed and working.

image

**Comment by dnnrly** [comment]
I don't suppose it's possible to create a different compose file and let people choose between them? Then we don't need a different branch, just document which one is needed.

File: docker-compose.yaml [hunk 1]
```diff
@@ -10,23 +10,7 @@ services:
...
-    command: nvidia-smi
-    deploy:
-      resources:
-        reservations:
-          devices:
-            - driver: nvidia
-              count: 1
-              capabilities: [gpu]
-
+      - ./open-webui:/Users/james/.ollama/models
```
Comment (dnnrly) [inline docker-compose.yaml]: Can we make this a relative path?

**Comment by jameskingtech** [comment]
> I don't suppose it's possible to create a different compose file and let people choose between them? Then we don't need a different branch, just document which one is needed.

Yeah I can sort that out now!

**Comment by jameskingtech** [comment]
@dnnrly should be good for you to review + merge when happy. 

I've also added some instructions to the README at the bottom to hopefully make life a bit easier for anyone using it!

**Comment by jameskingtech** [comment]
Ohh wait, whoops, I've still go the original docker-compose deleted. Let me put that back 😅

**Comment by jameskingtech** [comment]
There we go, good to go! 😄

**Review by dnnrly — APPROVED** [review]

**Pull request merged**